test/spot/public.test.ts

```diff
@@ -0,0 +1,57 @@
+import {SpotClient} from "../../src/spot-client";
+import {successResponseListSpot, successResponseObjectSpot} from "../response.util";
```

Comment:
```suggestion
import { SpotClient } from "../../src/spot-client";
import { successResponseListSpot, successResponseObjectSpot } from "../response.util";
```

nit: padding around braces